src/GitHub.Logging/Logging/Log.cs

```diff
@@ -0,0 +1,88 @@
+            => Logger.Value.Debug(messageTemplate, propertyValues);
+
+        public static void Debug(Exception exception, string messageTemplate)
+            => Logger.Value.Debug(exception, messageTemplate);
+
+        public static void Debug(Exception exception, string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Debug(exception, messageTemplate, propertyValues);
+
+        public static void Error(string messageTemplate)
+            => Logger.Value.Error(messageTemplate);
+
+        public static void Error(string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Error(messageTemplate, propertyValues);
+
+        public static void Error(Exception exception, string messageTemplate)
+            => Logger.Value.Error(exception, messageTemplate);
+
+        public static void Error(Exception exception, string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Error(exception, messageTemplate, propertyValues);
+
+        public static void Fatal(Exception exception, string messageTemplate)
+            => Logger.Value.Fatal(exception, messageTemplate);
+
+        public static void Fatal(Exception exception, string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Fatal(exception, messageTemplate, propertyValues);
+
+        public static void Fatal(string messageTemplate)
+            => Logger.Value.Fatal(messageTemplate);
+
+        public static void Fatal(string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Fatal(messageTemplate, propertyValues);
+
+        public static void Verbose(string messageTemplate)
+            => Logger.Value.Verbose(messageTemplate);
+
+        public static void Verbose(string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Verbose(messageTemplate, propertyValues);
+
+        public static void Verbose(Exception exception, string messageTemplate)
+            => Logger.Value.Verbose(exception, messageTemplate);
+
+        public static void Verbose(Exception exception, string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Verbose(exception, messageTemplate, propertyValues);
+
+        public static void Warning(string messageTemplate)
+            => Logger.Value.Warning(messageTemplate);
+
+        public static void Warning(string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Warning(messageTemplate, propertyValues);
+
+        public static void Warning(Exception exception, string messageTemplate)
+            => Logger.Value.Warning(exception, messageTemplate);
+
+        public static void Warning(Exception exception, string messageTemplate, params object[] propertyValues)
+            => Logger.Value.Warning(exception, messageTemplate, propertyValues);
+
+        public static void Assert(bool condition, string messageTemplate)
```

Comment:
This `Assert` appears to be the only method in this class that is referenced by anything. Are we planning to use the `Log` class in future or could be it be simplified/removed?

![image](https://user-images.githubusercontent.com/11719160/32369161-200270de-c080-11e7-97d8-1b456290a86b.png)

Reply:
I was wondering about this myself. @StanleyGoldman originally wrote this class I believe, lets see what he thinks?

Reply:
Yea, the `Assert` methods were the only ones that were needed. I added the other overloads in case there were other scenarios where you wanted to log but it was awkward to create a logger. But if they really aren't used, we can just kill the other methods.

Reply:
Yeah, I say lets kill them for now and add them back in if we need to.

Reply:
Yeah, I say lets kill them for now and add them back in if we need to.